src/Raven.Server/Smuggler/Documents/CsvStreamSource.cs

```diff
@@ -20,6 +21,28 @@
 
 namespace Raven.Server.Smuggler.Documents
 {
+    public class CsvConfiguration
```

Comment:
CsvImportOptions (to match naming from smuggler)

Reply:
done